models/voteRecords.ts

```diff
@@ -17,8 +17,16 @@ export function isYesVote(voteRecord: VoteRecord) {
         case VoteKind.Deny: {
           return false;
         }
-        default:
-          throw new Error('Invalid voteKind');
+        case 2 as any /*VoteKind.Abstain*/: {
+          return false;
+        }
+        case 3 as any /*VoteKind.Veto*/: {
+          return false;
+        }
```

Comment:
how come voting was working before without that??

Reply:
it break the app when you load a governance that had experienced one of the vote. Got it break on FakeDAO